### ISSUE TITLE
Display two Sounds in Notification if the confidence level is <0.70 and >0.50.

### ISSUE THREAD

**Issue by akkaneror** [makeabilitylab/SoundWatch#12]
As discussed on Slack, DJ and I think it might be helpful if we could display two sounds instead of one if 0.5 < Confidence Level < 0.7.
For the condition that trigger this behavior, DJ first suggested it would be something like this:
1. All sounds are below 0.5. Don’t show anything
2. Top sound is between 0.5 and 0.7. Show the second top sound as well (if it’s greater than 0.5)
3. Top sound is above 0.7, only show the top sound.

For the UI, we also need to think about **How** should we display two sounds unobtrusively (since the watch notification is small), and how Snoozing options would work in that case. There are several options that I suggest:
1. the Higher confidence sound is to be on top, followed by the lower confidence sound + a suggestive word like "maybe, or, etc..":
**Speech** (maybe **Baby Cry**)
This option allow us to not change the behavior of the snoozing button as well as the selection menu
2. Display two sounds equally. Having one separated button for each sound. So there would be three button in total: "Snooze Speech for 10 min", "Snooze Baby Cry for 10 mins" and "Open." We also need to rethink on how to tackle the selection menu, since right now, we have full screen snooze option that only snooze one sound at a time. 
![image](https://user-images.githubusercontent.com/42983232/91372887-be937900-e7c9-11ea-9208-5a6d972d0af5.png)

**Comment by hungngo97** [comment]
What if the top sound is 0.6 but the next ones accuracy is only 0.1, 0.1, 0.1, etc then the 2nd sound is pretty much noise. Personally I think another filter like of the 2nd sound is at least 0.2 or 0.3, or even we can do the initial filter like if 1st sound + 2nd sound accuracy > 0.9 then we display 2 sounds

**Comment by akkaneror** [comment]
> What if the top sound is 0.6 but the next ones accuracy is only 0.1, 0.1, 0.1, etc then the 2nd sound is pretty much noise. Personally I think another filter like of the 2nd sound is at least 0.2 or 0.3, or even we can do the initial filter like if 1st sound + 2nd sound accuracy > 0.9 then we display 2 sounds

Yeah, he said Top sound is between 0.5 and 0.7. Show the second top sound as well (if it’s greater than 0.5)
